src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -99,29 +94,29 @@ public IndexSubsystem(WPI_TalonFX firstMotor, WPI_TalonFX secondMotor, DigitalIn
                 .withSize(2, 1)
                 .getEntry();
 
-        this.ingestMotor = firstMotor;
-        this.feederMotor = secondMotor;
-        this.ingestProximity = ingestProximity;
-        this.feederProximity = feederProximity;
-        this.ingestTopProximity = ingestTopProximity;
-        this.ingestBlueColor = ingestBlueColor;
-        this.ingestRedColor = ingestRedColor;
-        this.feederBlueColor = feederBlueColor;
-        this.feederRedColor = feederRedColor;
-        this.ingestTopBlueColor = ingestTopBlueColor;
-        this.ingestTopRedColor = ingestTopRedColor;
+        ingestMotor = new WPI_TalonFX(INDEX_INGEST_MOTOR);
+        feederMotor = new WPI_TalonFX(INDEX_FEEDER_MOTOR);
+        // ingestProximity = new DigitalInput(INGEST_PROXIMITY);
+        feederProximity = new DigitalInput(FEEDER_PROXIMITY);
+        ingestTopProximity = new DigitalInput(INGEST_PROXIMITY);
+        // ingestBlueColor = new DigitalInput(INGEST_BLUE);
+        // ingestRedColor = new DigitalInput(INGEST_RED);
+        // feederBlueColor = new DigitalInput(FEEDER_BLUE);
+        // feederRedColor = new DigitalInput(FEEDER_RED);
+        // ingestTopBlueColor = new DigitalInput(INGEST_TOP_BLUE);
+        // ingestTopRedColor = new DigitalInput(INGEST_TOP_RED);
 
-        this.feederMotor.setInverted(true);
-        this.ingestMotor.configFactoryDefault();
-        this.feederMotor.configFactoryDefault();
+        feederMotor.setInverted(true);
```

Comment:
While you're here, delete this since it's set again on line 119 (likely because it's wiped out on configFactoryDefault below)